project-birth/CONTRIBUTING-DURING-START.md

```diff
@@ -52,7 +59,7 @@ Throughout the documentation there are many references to other services, and we
 When these are encountered we a few options:
 
 1. Find the relevant equal service. Such as finding linkage to the `Atom` or `Atom-Community` Discord Server, could be updated to instead point to our Discord Server, with the branding changed to be relevant and accurate.
```

Comment:
```suggestion
1. In the atom-archive folder leave these as is. After moving a copy to it's relevant section in the docs: 
  - Find the relevant equal service. Such as finding linkage to the `Atom` or `Atom-Community` Discord Server, could be updated to instead point to our Discord Server, with the branding changed to be relevant and accurate.
```

Fixed this for the part where we're leaving the atom-archive contents as is beyond making them functional. And only modifying after we've moved them to their relevant updated section.

Reply:
I'm not sure this will fit properly, since this is still talking about rebranding within the code, as in think of the rfc docs within our Pulsar branch. Now at this point we should probably have a section dedicated to documentation as a whole, in which having this bit would make most sense. Maybe we save this for later until that other section is written out?

Reply:
I mean, my goal in mentioning this was putting it out there that when it's in the archive folder not to change anything. But once it's moved, change to whatever is needed at that point. Since we want to make sure it's as accurate as possible to the original in its original location, but change what we need in its final location.

Reply:
Fair enough, I'll see about adding a section for specifically documentation that will cover this then in anther PR.

Otherwise what do we think? With approval we can probably try and get this merged.

Reply:
I mean yea, I think I misread that original one. But it should be clear I guess from the original wording. Do want that to be clear going forward to keep an archived version somewhere and update it in a new location that's appropriate. Since that should be mentioned somewhere at some point.